x/fungible/keeper/gas_coin_and_pool.go

```diff
@@ -15,10 +15,10 @@ import (
 	zetaObserverTypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
-// setup gas ZRC20, and ZETA/gas pool for a chain
+// SetupChainGasCoinAndPool setup gas ZRC20, and ZETA/gas pool for a chain
 // add 0.1gas/0.1wzeta to the pool
 // FIXME: use chainid instead of chain name; add cointype and use proper gas limit based on cointype/chain
```

Comment:
NIT : Lets modify the TODO here , as it uses chain ID now

Reply:
2d30a948